src/components/@common/Recommendations/User/UserCard/index.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * 친구 찾아보기 - 유저 카드
+ * @author 선우
+ */
+
+import PersonAdd from '@/components/@common/SVG/Icon/PersonAdd';
+import { IRecommendedUsers } from '@/types/@common';
+import { ReactEventHandler, useState } from 'react';
+import * as Styled from './UserCard.styled';
+import defaultProfileImg from '@/assets/images/defaultProfile.png';
+
+export interface IUserCardProps {
+  userInfo: IRecommendedUsers['content'][0];
+}
+
+const UserCard = ({ userInfo }: IUserCardProps) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  const handleImgError: ReactEventHandler<HTMLImageElement> = (e) => {
+    e.currentTarget.src = defaultProfileImg;
+  };
+
+  // TO-DO: 친구 추가 API 연동
+  const handleButtonClick = () => {
+    console.log('친구 추가');
+  };
+
+  return (
+    <Styled.UserCardWrapper
+      onMouseOver={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
+    >
+      <Styled.UserCardIntroSection>
+        <Styled.UserCardImage
+          src={userInfo.imageUrl}
+          onError={handleImgError}
+        />
+        <Styled.UserCardInfoSection>
+          <Styled.UserCardNickname>{userInfo.nickname}</Styled.UserCardNickname>
+          <Styled.UserCardDescription $isHovered={isHovered}>
```

Comment:
```suggestion
          <Styled.UserCardDescription>
```
$isHovered props를 넘겨주지 않고, css에서 아래처럼 작성해도 좋을 것 같습니다.

```
&:hover {
    text-overflow: ellipsis;
    width: 55%;
  }
```

Reply:
`UserCardDescription` 은 span 태그라 span에 의사 클래스를 붙이게 되면
**span에 hover가 될 때만** 말줄임표가 적용 되어서요...!

유저 카드 전체에 hover 됐을 때 말줄임표가 생기도록 `$isHovered` prop을 사용해서 조건부로 구현했습니다!

혹시 제가 잘못 이해한 부분이 있다면 알려주세요 😃

Reply:
아 그렇군요...! 그럼 선우님 말씀대로 구현하는 것이 좋겠네요👍

<img width="482" alt="image" src="https://github.com/user-attachments/assets/2bdddc32-a944-41bb-94a4-0b2ac53c1c44" />

테스트 해보다가 발견한 점인데, selfDescription이 길어졌을 때 li 태그 padding 안쪽까지 글자가 존재해야 할 것 같습니다...!
이 부분만 확인 부탁드립니다!

Reply:
말씀 해주신 부분 수정했습니다! 꼼꼼히 봐주셔서 감사드려요 😄

Reply:
친구신청 버튼과 친구 info 사이에 간격이 존재하는 것 같습니다! UserCardDescription 스타일에 padding-right: 40px;이 추가되어야 할 것 같아요🙏

<img width="483" alt="스크린샷 2025-02-12 오후 12 48 23" src="https://github.com/user-attachments/assets/75608f34-4e69-41a9-abfa-dda7079922b0" />

Reply:
혹시 `padding-right: 40px;` 이 어디에 적용되어 있는지 알 수 있을까요?
디자인 모드에서는 (프로필 이미지 + 닉네임 + 자기소개) / 친구신청 버튼이 **space-between**으로만 되어 있어서요..!

Reply:
<img width="462" alt="image" src="https://github.com/user-attachments/assets/6b259665-b295-4b94-ab0a-dabcc94d767e" />

현재는 위처럼 구현되어 있는데, 아래 디자인처럼 gap이 필요해보여서요!
padding-right: 40px;는 게시판쪽 친구 찾아보기 색션을 보았네요. 죄송합니다🙏

<img width="809" alt="스크린샷 2025-02-13 오후 7 57 43" src="https://github.com/user-attachments/assets/96d95b86-b790-44c9-a924-420f46ce3281" />

Reply:
![image](https://github.com/user-attachments/assets/1fc7c9be-06b7-4bef-a06c-32073bba4937)

수정했습니다!